.env

```diff
@@ -67,3 +67,7 @@ HYKU_RESTRICT_CREATE_AND_DESTROY_PERMISSIONS=true
 # AND comment this out
 HYRAX_ANALYTICS=false
 ##END## Enable Google Analytics
+
+# When set to false, this turns off turbolinks which allows for
+# redirects from AF works to Valyrie works in the edit and show
+HYKU_ALLOW_REDIRECT_TO_VALKYRIE_RESOURCE=true
```

Comment:
Non-blocker, but curious why not name this `HYKU_BLOCK_VALKYRIE_REDIRECT`?  There's a method that toggles the truthiness of this value and then we used that toggled value.

Reply:
That sounds good to me, I can make the change